src/Tables.jl

```diff
@@ -116,7 +118,8 @@ Obviously every table type is different, but via a combination of `Tables.rows`
 abstract type Table end
 
 # default definitions
-istable(x::T) where {T} = istable(T)
+istable(x::T) where {T} = istable(T) || TableTraits.isiterabletable(x) === true ||
+    TableTraits.isiterabletable(x) === missing
```

Comment:
Is this a good idea? That essentially means that any consumer of ``istable`` must now be prepared that even if ``istable`` returns ``true``, that it actually is not a table...